tools/clang/lib/SPIRV/SpirvEmitter.h

```diff
@@ -122,6 +122,10 @@ class SpirvEmitter : public ASTConsumer {
                                SourceRange rangeOverride = {});
   SpirvInstruction *doCompoundAssignOperator(const CompoundAssignOperator *);
   SpirvInstruction *doConditionalOperator(const ConditionalOperator *expr);
+  SpirvInstruction *doConditional(const Expr *expr,
+                                  const Expr *cond,
+                                  const Expr *falseExpr,
+                                  const Expr *trueExpr);
```

Comment:
this could remain private to SpirvEmitter no?

Reply:
It can and I believe it is (see line 95 for the `private` label)

Reply:
Yes! Sorry, saw the header and assumed it was public.